torch_patches/X10-xla-tensor-storage.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/aten/src/ATen/core/TensorBase.h b/aten/src/ATen/core/TensorBase.h
+index e6dd7365..c80b7dc0 100644
+--- a/aten/src/ATen/core/TensorBase.h
++++ b/aten/src/ATen/core/TensorBase.h
+@@ -326,7 +326,7 @@ class TORCH_API TensorBase {
+     return impl_->storage();
+   }
+   bool is_alias_of(const at::TensorBase& other) const{
+-    return impl_->storage().is_alias_of(other.storage());
++    return is_xla() ? false : impl_->storage().is_alias_of(other.storage());
```

Comment:
This won't be the proper fix. Either we integrate fully with functionalization or we implement something like https://github.com/pytorch/pytorch/pull/75246 as a transition plan. The former requires us to adopt LTC, and therefore I'm going to introduce the latter to XLA.